packages/uniforms/src/types.ts

```diff
@@ -50,13 +50,18 @@ export type GuaranteedProps<Value> = {
   id: string;
   label: ReactNode;
   name: string;
-  onChange(value?: Value | null, name?: string): void;
+  onChange: OnChange<Value | undefined>;
```

Comment:
This change doesn't take into consideration the previous `null` typing. Is this intended? (I see the `null`s were changed into `undefined`s but just wanted to make sure.

Reply:
Yes, it was intentional. I wrote `null` here as I prefer them over `undefined`, but none of the `onChange` handlers have had special `null` handling.